### ISSUE TITLE
Cyrillic fonts for path directory

### ISSUE THREAD

**Issue by dragon1ce** [PathOfBuildingCommunity/PathOfBuilding#7364]
### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### Is your feature request related to a problem?

Yes,Path of building cannot install in c because the username for the pc is in cyrillic and it just shows ????? instead of the name of the folder.I installed it on D but cannot save anything because it says cannot save settings.xml in the directory.

### Describe the solution you'd like

UTF-8

### Describe alternatives you've considered

Giving me a tip how to change default save for the settings xml.
Because when i try to change directory for builds it says error cannot save settings.xml and without it cannot save builds with the changeg directory path.

### Additional context

Its not really important but if its a easy fix,or you have a suggestion how to help myself ill try :)

**Comment by Paliak** [comment]
You have a few options. From a user's perspective using the standalone version of POB would probably be easiest as it will save settings.xml to the location of the folder containing POB. That said if you'd like to keep using the installed version you'll probably have to edit https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/a93ebb2524c791e92fe4bf0ac954a095ef73b2f4/src/Modules/Main.lua#L53-L59

`GetUserPath()` returns the value of `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders` which by default should be `%USERPROFILE%\Documents`. This means that you can either edit the registry entry but that will likely affect other programs or try changing the `%USERPROFILE%` env var for POB. https://superuser.com/questions/424001/launch-windows-program-with-custom-environment-variable

**Comment by Nightblade** [comment]
@dragon1ce Did you resolve this issue?

**Comment by dragon1ce** [comment]
Yes but reinstalling my pc with english user name so english file directory
